backend/api/views.py

```diff
@@ -192,7 +192,7 @@ def post(self, request):
             request.user.profile.entry_form_data = {}
             request.user.profile.save()
 
-            return Response({"sac_id": sac.id, "next": "TBD"})
+            return Response({"report_id": sac.report_id, "next": "TBD"})
```

Comment:
[NB] Should "next" still exist? Or point to whatever the screen to review it is or something? Or should it still be TBD since an actual future step is undetermined?

Reply:
Fair question. As of now this doesn't do anything (clearly, since "TBD") because we're letting the front-end determine where to go next. I can imagine a world where we'd prefer to have that in the DB instead.